### DISCUSSION TITLE
Could not connect to X session manager: Authentication Rejected, reason : None of the authentication protocols specified are supported and h

### DISCUSSION THREAD

**Discussion by War3Evo** [canonical/steam-snap#117] [General]
I think every time steam tries to do a bottom right pop-up notification, and it doesn't work correctly and disappears quickly, this error message pops up:

Could not connect to X session manager: Authentication Rejected, reason : None of the authentication protocols specified are supported and h

Debian 11 XFCE

**Comment by marclundgren** [comment]
@War3Evo any update on this issue? Did you ever overcome it?  I'm running into the same problem with steam on flatpak
